src/vellum/workflows/nodes/core/try_node/node.py

```diff
@@ -160,6 +160,7 @@ class Outputs(inner_cls.Outputs):  # type: ignore[name-defined]
                     "__module__": dynamic_module,
                     "on_error_code": _on_error_code,
                     "subworkflow": Subworkflow,
+                    "Ports": type("Ports", (TryNode.Ports,), {port.name: port.copy() for port in inner_cls.Ports}),
```

Comment:
This is the key line of the PR - we copy over all of the ports from the wrapped node to the adornment